languages/java8/template/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+    </properties>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.algorithmia</groupId>
+    <artifactId>algorithm</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>com.algorithmia</groupId>
+            <artifactId>algorithmia-client</artifactId>
+            <version>[1.0.11,)</version>
+        </dependency>
+        <dependency>
+            <groupId>com.algorithmia</groupId>
+            <artifactId>algorithm-handler</artifactId>
+            <version>[1.1,)</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
Junit should be marked as a test only dependency

Reply:
There doesn't seem to be a clean way to do this with maven which won't add additional maven/pom complexity, do you have an alternative pom structure in mind?